packages/cli/create/template/apps/site/handler/handler.js

```diff
@@ -2,6 +2,7 @@ const fs = require("fs");
 const path = require("path");
 const mime = require("mime-types");
 const isUtf8 = require("isutf8");
+const zlib = require('zlib');
```

Comment:
This looks like you've committed without running the commit hooks.  Please either format the file with prettier, or let the commit hooks do it for you. But don't use the `--no-verify` ;)

Reply:
done